x-pack/plugins/apm/public/components/shared/charts/helper/get_alert_annotations.test.tsx

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_SEVERITY_LEVEL } from '@kbn/rule-data-utils/target/technical_field_names';
+import { ValuesType } from 'utility-types';
+import { EuiTheme } from '../../../../../../../../src/plugins/kibana_react/common';
+import { ObservabilityRuleTypeRegistry } from '../../../../../../observability/public';
+import { APIReturnType } from '../../../../services/rest/createCallApmApi';
+import { getAlertAnnotations } from './get_alert_annotations';
+
+type Alert = ValuesType<
+  APIReturnType<'GET /api/apm/services/{serviceName}/alerts'>['alerts']
+>;
+
+const euiColorDanger = 'red';
+const euiColorWarning = 'yellow';
+const theme = ({
+  eui: { euiColorDanger, euiColorWarning },
+} as unknown) as EuiTheme;
+const alert: Alert = {
+  'rule.id': ['apm.transaction_duration'],
```

Comment:
fwiw, there are constants for these (in @kbn/rule-data-utils)

Reply:
I see you've already figured that out 🤷🏻‍♂️

Reply:
Yes this is just a test https://www.youtube.com/watch?v=aKSNnaVIRj8